### ISSUE TITLE
[Feature] A way to delete a schema tag

### ISSUE THREAD

**Issue by jhalborg** [apollographql/apollo-tooling#948]
I think it'd be great if it was possible to remove schema tags - or maybe it exists, and I'm just blind? 

A `current` tag is automatically created in Engine when creating a service, and is set as default. I'd like to remove it, and instead only use our `next`, `stage`, and `production` tags, removing `current` al together. 

As far as I can tell, this is not possible at the moment?

**Comment by JakeDawkins** [comment]
@jhalborg you're right! Right now, there's no way to delete tags. I don't know about timelines, but this will exist eventually

**Comment by JakeDawkins** [comment]
@jgzuke will be looking into this soon 🎉
